src/app/events/page.tsx

```diff
@@ -240,15 +246,31 @@ export default function Events({ searchParams }: Props) {
         kicker="Get Involved"
         title="Get in Touch With the Events Team"
       >
-        <CardGrid cols="mdThree">
-          {getInvolvedData.map(({ title, description, cta }) => (
-            <Card
-              key={title}
-              title={title}
-              description={description}
-              cta={cta}
+        <CardGrid cols="mdTwo">
```

Comment:
Would you say this is a list with list items?

Reply:
✅